src/fides/api/ops/schemas/privacy_experience.py

```diff
@@ -20,7 +20,7 @@ class PrivacyExperience(BaseSchema):
     disabled: Optional[bool] = False
     component: ComponentType
     delivery_mechanism: DeliveryMechanism
```

Comment:
I think we may need this to be optional, since if the component type is `privacy_center`, we don't bother with a delivery mechanism

Reply:
That will work better with some of the things I'm seeing in Michael's figma anyway! but question, the `delivery_mechanism` isn't link in that case?

Reply:
yeah I was wondering if that would be the case—thinking about it more, I guess it certainly can be, although there isn't a field in the form for the mechanism if it's privacy center. so I could just as easily always pass `link` if the component type is `privacy_center`?

Reply:
I was assuming you'd pass in `link` here, but I may need to make `delivery_mechanism` optional regardless if we end up having to save mostly empty privacy experiences when privacy notices are created/updated to be further updated on the experience UI. I'm still not sure of the best workflow there.